inc/class-wpseo-rank.php

```diff
@@ -138,11 +138,11 @@ public function get_label() {
 	 */
 	public function get_drop_down_label() {
 		$labels = array(
-			self::NO_FOCUS => __( 'SEO: No Focus Keyphrase', 'wordpress-seo' ),
-			self::BAD      => __( 'SEO: Needs improvement', 'wordpress-seo' ),
-			self::OK       => __( 'SEO: OK', 'wordpress-seo' ),
-			self::GOOD     => __( 'SEO: Good', 'wordpress-seo' ),
-			self::NO_INDEX => __( 'SEO: Post Noindexed', 'wordpress-seo' ),
+			self::NO_FOCUS => sprintf( __( 'SEO: %s', 'wordpress-seo' ), __( 'No Focus Keyphrase', 'wordpress-seo' ) ),
```

Comment:
Please add translator comments about the %s like in the files above.

Reply:
Done.